admin.php

```diff
@@ -935,4 +935,4 @@ function vsram(){
 
 } // Class Ends
 
-?>
\ No newline at end of file
+?>
```

Comment:
The GitHub editor seems to be adding this new line, will clone to desktop in future.